src/components/tooltips-hoc/DropPod.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as _ from 'underscore';
+import {positionValidator} from './positionValidation';
+
+export const DropPodPosition = {
+    bottom: 'bottom',
+    top: 'top',
+    left: 'left',
+    right: 'right',
+};
+
+export interface IDropPodProps {
+    isOpen: boolean;
+    positions?: string[];
+    buttonRef: React.RefObject<HTMLElement>;
+    renderTooltip: (style: React.CSSProperties, containerRef: React.RefObject<HTMLElement>) => React.ReactNode;
+    selector: string;

```

Comment:
This should be optional since you have a default

```suggestion
    selector?: string;
```